llvm/lib/SYCLLowerIR/ComputeModuleRuntimeInfo.cpp

```diff
@@ -188,6 +191,10 @@ PropSetRegTy computeModuleProperties(const Module &M,
   if (GlobProps.EmitExportedSymbols) {
     // extract exported functions if any and save them into property set
     for (const auto *F : EntryPoints) {
+      // Virtual functions use different mechanism of dynamic linking, they
```

Comment:
```suggestion
      // Virtual functions use a different mechanism of dynamic linking, they
```

Reply:
Applied in 876c8a22f8c12191d4d5b6361ef9c826153c226a